packages/bot-web-ui/src/components/dashboard/tutorial-tab/guide-content.tsx

```diff
@@ -60,30 +61,21 @@ const GuideContent = ({
             <div className='tutorials-wrap__group'>
                 {finalContentArray.length > 0 ? (
                     finalContentArray.map(items => {
```

Comment:
Can we use desctructurization here?
```suggestion
                    finalContentArray.map({ id, content, url = '' } => {
```

Reply:
done ✅